src/Event.php

```diff
@@ -4,6 +4,8 @@
 
 namespace Sentry;
 
+use Jean85\PrettyVersions;
```

Comment:
```suggestion
```

Reply:
This is used, cannot be removed. I removed the line after this one though.